bemani/backend/ddr/ddrsn2/__init__.py

```diff
@@ -0,0 +1,3 @@
+from bemani.backend.ddr.ddrsn2.ddrsn2 import DDRSuperNova2
```

Comment:
Interesting separating this into a module instead of a single file. Any reason for having this across a bunch of files?

Reply:
Honestly, mostly just a leftover from my JS work. A common lint is only one class defined per file. If it's a bit awkward I'm ok with merging them into one file.

Reply:
IDGAF either way, pattern in the repo is one file, we don't do silly things like have an import system that depends on there being one export/class per file, so you can get away with a lot of stuff.